docker/Dockerfile

```diff
@@ -216,6 +228,25 @@ RUN set -e; \
     esac; \
     truncate -s0 ~/.bash_history;
 
+RUN set -e; \
+    if [ -z $(which bazel) ]; then \
+        BAZEL_INSTALL_DIR='/usr/local/bin/'; \
```

Comment:
Don't do this without packaging. Use fpm

Reply:
Reasons and maybe sample commands, please?

Reply:
- Keep your files well-tracked.
- Check for conflicts to avoid accidentally overwrite others without knowing.
- Carries metadata.
- Allow people to understand what's installed without knowing how it was initially built.
- Allow uninstallation.

IIRC you've used fpm to package code for deployment.
If not see https://github.com/xkszltl/Roaster/blob/master/pkgs/utils/fpm/install.sh and https://github.com/xkszltl/Roaster/blob/aa2fc8f56dbcf4405662a13daf3ce3b9e5ddcd09/pkgs/utils/fpm/install_from_git.sh#L33-L50 for sample command.

If built by yourself, you can also write a script like this to rely on roaster to pack it for you:
https://github.com/xkszltl/Roaster/blob/master/pkgs/glog.sh